packages/vaadin-grid/src/vaadin-grid.d.ts

```diff
@@ -2,7 +2,7 @@ import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.
 
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
-import { GridDropLocation, GridItem, GridItemModel } from './interfaces';
+import {DefaultGridItem, GridDropLocation, GridItemModel} from './interfaces';
```

Comment:
Seems that the object definitions have not formatted by Prettier.

Reply:
Yep, ATM the prettier / lint-staged config does not cover Typescript files, so none of them are formatted correctly currently. I thought about adding this here but I decided to not change this in this PR in order to not create more clutter that would distract from the actual changes to the types.

I was thinking about opening a separate PR for that later on.